src/api/app/models/history_element/request_review_added.rb

```diff
@@ -5,6 +5,8 @@ def description
     end
 
     def user_action
+      return 'added a reviewer' if review.nil?
```

Comment:
This is needed for the old request view which calls `user_action` independently. If review is not present this would have crashed in the old view too.